Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1381,9 +1381,12 @@ static AdditionalFonts additionalFonts(const Vector<URL>& fontURLs, std::optiona
 
         Vector<URL> fontURLs;
         for (NSString *font in enumerator.get()) {
-            if ([font hasSuffix:@"ttf"] || [font hasSuffix:@"ttc"] || [font hasSuffix:@"otf"]) {
-                RetainPtr fontsPath = [userInstalledFontsPath stringByAppendingPathComponent:font];
-                auto fontURL = URL::fileURLWithFileSystemPath(String(fontsPath.get()));
+            RetainPtr fontsPath = [path stringByAppendingPathComponent:font];
+            auto fontURL = URL::fileURLWithFileSystemPath(String(fontsPath.get()));
+            NSURL *nsFontURL = fontURL;
+            if ([nsFontURL.pathExtension caseInsensitiveCompare:@"ttf"] == NSOrderedSame
```

Comment:
Is there a way to delegate extension checking to underlying frameworks, so that we'd be looking at UTIs?

Reply:
That is a very good point! I will look closer at that.

Thanks for reviewing!